pyscriptjs/tests/integration/test_py_repl.py

```diff
@@ -674,3 +674,47 @@ def test_repl_src_path_that_do_not_exist(self):
             "Are your filename and path correct?"
         )
         assert self.console.error.lines[-1] == errorMsg
+
+    @skip_worker("dont-care")
+    def test_repl_results(self):
+        self.writefile("loadReplSrc2.py", "2")
+        self.writefile("loadReplSrc3.py", "print('3')")
+
+        self.pyscript_run(
+            """
+            <py-repl id="py-repl1" output="out1">
+            42
+            </py-repl>
+            <div id="out1"></div>
+
+            <py-repl id="py-repl2" output="out2">
+            c = [1,2,3]
+            from sys import getrefcount
+            # should print 2: 1 from the reference c and 1 since getrefcount
+            # holds a reference to its argument
+            print(getrefcount(c))
+            c
+            </py-repl>
+            <div id="out2"></div>
+
+            <py-repl id="py-repl3" output="out3">
+            # should also print 2: if it prints 3 that would mean that c was not properly
```

Comment:
dare I say this should be observable ... leaking `getrefcount(c)` globally in the previous *repl* and being sure here it's `previous_count - 1` ??? not sure who's going to look at the build output otherwise for this detail.

Reply:
never mind, I see this is checked later on via console ... resolving and approving!